### ISSUE TITLE
[a11y Support] Benefits Non-Disability Experience Office Hours Follow-Up: Wizard and Slideshow

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#73590]
# Office Hours Follow-Up Information and Links
- **Session**: 1/9/2024
- **Attended** **by**: Fiorella Glavez

## Wizards
- **Unified Wizard Guidance**
    - Buried on the [deprecated page is some guidance on how to retro fit the pattern](https://design.va.gov/patterns/wizards#retrofit-this-pattern)
        - This guidance was written by David Kennedy aka DK - an a11y specialist
        - It should be `"all" wizards that are deprecated`.

## Conditionally Revealed
- **Conditionally revealed isn't considered a wizard**
    - That [guidance for conditionally revealed is buried here](https://design.va.gov/patterns/ask-users-for/relationship#conditionally-revealed-fields-can-be-used-if-the-following-conditions-are-met).
    - So, that might seem like a wizard, but it's pretty controlled in that it's one field being revealed and has some strict rules around it.
    - It doesn't have to be tied to relationship to a veteran either, that's just the example case.
- **Ideally** - wizards turn more into one concept per page with conditional routing.

## Missing Progress Bar
- If it's `pre-screener,` it doesn't have steps, it's only to help figure out if it is a form they should fill out. I
- Jeana said: I don't think there's guidance anywhere about that, so it could be something worth discussing.
